recipes/args/all/conanfile.py

```diff
@@ -0,0 +1,108 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class ArgsConan(ConanFile):
+    name = "args"
```

Comment:
Args is a very common name, can you please put it in a namespace?
See https://github.com/conan-io/conan-center-index/pull/3233#discussion_r506982833

Reply:
Done, will be part of upcoming patch.